examples/gno.land/p/demo/dao/vote.gno

```diff
@@ -0,0 +1,70 @@
+package dao
```

Comment:
Add a comment to explain that the vote will be removed from this p/demo/dao package. A DAO shouldn't have to comply with or define how the voting mechanism works internally; it should be viewed as an entity that makes decisions. Therefore, we should focus on the proposal rather than the votes.

Reply:
I want to merge this PR soon, so I'm in favor of just adding the comment. However, please note that this is the biggest weakness of this implementation, and it's the only issue I see that will require us to create a v2.

Reply:
I checked, it turns out the extent of "votes" being present in the dao package is just in the context of a single type `Stats`. We don't enforce a voting strategy in the `p/demo/dao` package, at all. 

`simpledao`, which is a DAO implementation, exposes its own vote method, that is not standard.

I've added a comment to make it clear:

[ee989ff](https://github.com/gnolang/gno/pull/2581/commits/ee989ff12115156b2bed326766a510110abe3260)